Doc/c-api/init_config.rst

```diff
@@ -828,6 +828,23 @@ PyConfig
 
       Default: ``0``.
 
+   .. c:member:: int int_max_str_digits
+
+      Configures the :ref:`integer string conversion length limitation
+      <int_max_str_digits>`.  ``-1`` means that
+      :data:`sys.int_info.default_max_str_digits` will be used.  No other
+      negative value is accepted.  ``0`` disables the limitation.  Values
+      greater than zero but less than ``_PY_LONG_MAX_STR_DIGITS_THRESHOLD``
+      (640) also known as :data:`sys.int_info.str_digits_check_threshold`
+      are unsupported and will produce an error.
+
```

Comment:
Maybe mention https://docs.python.org/dev/library/sys.html#sys.set_int_max_str_digits function as a "see also", but it may be redundant with <int_max_str_digits> link which documents it.

Reply:
I think that is redundant, the larger int_max_str_digits section link answers everything.